docs/content/Workspace/SQL-Runner.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: SQL Runner
+permalink: /cloud/sql-runner
+category: Workspace
+menuOrder: 6
+---
+
+The SQL Runner is a tool that allows you to execute SQL queries from Cube Cloud
+on your data source or Cube Store. It can be used for ad-hoc querying as well as
+debugging SQL queries generated by Cube to execute against the data source.
+
+<SuccessBox>
+
+The SQL Runner is available in Cube Cloud. [Click here][cubecloud-signup] to
+sign up for a new account.
+
+</SuccessBox>
+
+<Screenshot src="https://ucarecdn.com/6d443059-b990-4735-81c1-78a490aa2f25/" />
+
+## Executing queries
+
+To execute a query, enter the SQL query in the text area under **SQL Editor**
+and click <Btn>▶ Run</Btn>. The query results will be displayed under
+**Results**, along with the row count and query execution time:
+
+<Screenshot
+  highlight="inset(12% 2% 1% 41% round 10px)"
+  src="https://ucarecdn.com/2f910abf-5b94-43cd-87f8-14c86bd98f77/"
+/>
+
+## Using a Security Context
```

Comment:
Honestly, I struggle to understand how choosing a security context (which is relevant to queries incoming to Cube's APIs) is useful for queries against the data source. The explanation in this section don't really help. If I'm not completely out of mind here, we probably need to explain better 🙂